fbmsgbot/models/template.py

```diff
@@ -22,6 +33,10 @@ def __init__(self, type_, **kwargs):
 
         if self.type not in self.types:
             raise ValueError("Incorrect type param: ", self.type)
+
+        if self.type == self.receipt_type:
```

Comment:
@ahendy0 wot? these properties are just on the receipt template, not the element